### PR TITLE
Navigation component

### PR DESCRIPTION
**Pull request by darshandamre**
created navigation component
Preview [https://component-lib-git-navigation-darshandamre.vercel.app/pages/docs.html#navigation](https://component-lib-git-navigation-darshandamre.vercel.app/pages/docs.html#navigation)

### CODE REVIEW

**Review by ark2002 — APPROVED** [review]
looks good to me good to go

File: components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  max-width: 10rem;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.navigation__actions > ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (SRUJANA-PENUGONDA13) [inline components/navigation.css]: align-items center property already getting from it's parent container navigation_content after removing this here does not make any impact in your app (Verified in inspect)

File: components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  margin: 1rem auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation__logo {
+  max-width: 10rem;
+  display: flex;
+  align-items: center;
```
Comment (SRUJANA-PENUGONDA13) [inline components/navigation.css]: Same as does not making any impact after removing it (Verified in inspect)

File: styles/docs.css [hunk 3]
```diff
@@ -62,7 +63,8 @@ body {
 }
 
 .display-card--vertical-cards,
-.display-card--horizontal-cards {
+.display-card--horizontal-cards,
+.display-card--navigation {
   max-width: none;
```
Comment (SRUJANA-PENUGONDA13) [inline styles/docs.css:68]: This attribute as well.

**Comment by SRUJANA-PENUGONDA13** [comment]
Added few suggestions verify from your end once and remove if not requires. Everything looks good.

File: components/navigation.css (hunk 2 above)
Comment (darshandamre) [reply components/navigation.css]: removed these properties

File: styles/docs.css (hunk 3 above)
Comment (darshandamre) [reply styles/docs.css:68]: this is so that it takes as much as space available when it gets to larger screens, Thank you for your time.

**Pull request merged**